x/token/module/module.go

```diff
@@ -111,10 +111,14 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	token.RegisterQueryServer(cfg.QueryServer(), keeper.NewQueryServer(am.keeper, am.authKeeper))
 
 	// m := keeper.NewMigrator(am.keeper)
```

Comment:
How about replacing commented-out code for guidence with `NOTE: Place to add migrator`

Reply:
I thought the code would be out-dated, because we won't use the code now. I didn't want to maintain the unused logic, so I removed the entire logics.